ios/FluentUI/Navigation/Views/AvatarTitleView.swift

```diff
@@ -158,18 +147,32 @@ class LargeTitleView: UIView, TwoLineTitleViewDelegate {
         setupAccessibility()
         twoLineTitleView.delegate = self
 
-        NotificationCenter.default.addObserver(self,
-                                               selector: #selector(themeDidChange),
-                                               name: .didChangeTheme,
-                                               object: nil)
+        tokenSet.registerOnUpdate(for: self) { [weak self] in
+            self?.updateAppearance()
```

Comment:
Above, you setTitleColor after updateAppearance, but here you don't setTitleColor. That to me means either setTitleColor should be moved into updateAppearance, or also called here (preferably the former).

Reply:
Upon closer inspection, `updateAppearance` already calls `setTitleColor`, so we can take a third option of removing the extra call.